### PR TITLE
Place matching value props, advanced options

### PR DESCRIPTION
**Pull request by jsani-radar**
Per wg feedback, make the value props for place matching clearer, and add a section explaining the advanced options

### CODE REVIEW

File: docs/dashboard.mdx [hunk 1]
```diff
@@ -78,3 +78,9 @@ When an **isochrone** geofence is matched, the geofence will only adopt the cent
 When a **polygon** geofence is matched, the geofence will be linked to the Radar place, but not adopt the place center or geometry. This behavior supports geofences that represent areas such as parking lots and drive-thru lanes.
 
 ![Place matching polygon](/img/place-matching/place_matching_polygon.png)
+
+### Advanced options
+
+- **Search radius.** Place matching will look for the nearest location within 10 kilometers, consistent with our [search places](/api#search-places) endpoint. This value can be reduced if a tighter search window is desired.
+- **Geofence tag(s).** Filter the rows for which place matching is attempted. For example, place matching can only happen where the geofence tag is store, but not parking lot.
```
Comment (corypisano) [inline docs/dashboard.mdx:85]: maybe something like `For example, only perform place matching on rows where tag = "store" but not tag = "parking lot"`

File: docs/dashboard.mdx [hunk 2]
```diff
@@ -50,7 +50,7 @@ The User Activity page is the most straightforward way to debug a user's recent
...
+Place matching is the simplest way to make sure your geofences are accurate and stay up-to-date over time. Cleaning your geofence datasets via Radar's import process will reduce false positives by making sure that entry events are correct, trip approaching and arrived signals are improved, and messaging for nearby locations is relevant. Navigate to the [geofences](https://radar.com/dashboard/geofences) page and select *Import*. If [places](/places) are enabled for your project, opt-in to this feature by toggling on *Enable place matching*. Select [chains](/places/chains) to match your geofences against. The import process will attempt to match each geofence against Radar's places for those chains.
```
Comment (corypisano) [inline docs/dashboard.mdx:53]: nit: maybe "accurate" instead of improved for "trip approaching and arrived signals are ..."?

**Review by corypisano — APPROVED** [review]
Lgtm, some small and/or ignorable copy nits

File: docs/dashboard.mdx (hunk 2 above)
Comment (bschoeneweis) [inline docs/dashboard.mdx:53]: nit: also include exit events?

"will reduce false positives by making sure that entry and exit events are correct, trip approaching..."

**Review by bschoeneweis — APPROVED** [review]
Looks good.  Echoing @corypisano's comments as well but thanks for adding!

File: docs/dashboard.mdx (hunk 2 above)
Comment (berylw) [inline docs/dashboard.mdx:53]: nit: maybe specify what we mean by accurate? "...make sure your geofence coordinates and geometries are accurate and stay up-to-date over time"

File: docs/dashboard.mdx [hunk 3]
```diff
@@ -50,7 +50,7 @@ The User Activity page is the most straightforward way to debug a user's recent
 
 ## Place matching
```
Comment (berylw) [inline docs/dashboard.mdx:52]: wonder if we should do an upsell here with an info banner i.e. "Place matching requires that places are enabled for your project"

File: docs/dashboard.mdx [hunk 4]
```diff
@@ -78,3 +78,9 @@ When an **isochrone** geofence is matched, the geofence will only adopt the cent
 When a **polygon** geofence is matched, the geofence will be linked to the Radar place, but not adopt the place center or geometry. This behavior supports geofences that represent areas such as parking lots and drive-thru lanes.
 
 ![Place matching polygon](/img/place-matching/place_matching_polygon.png)
+
+### Advanced options
+
+- **Search radius.** Place matching will look for the nearest location within 10 kilometers, consistent with our [search places](/api#search-places) endpoint. This value can be reduced if a tighter search window is desired.
```
Comment (berylw) [inline docs/dashboard.mdx:84]: nit maybe something like `By default, place matching will look for the nearest place within 10 kilometers (the max radius provided by our search places endpoint).`"

File: docs/dashboard.mdx [hunk 5]
```diff
@@ -78,3 +78,9 @@ When an **isochrone** geofence is matched, the geofence will only adopt the cent
...
+- **Geofence tag(s).** Filter the rows for which place matching is attempted. For example, place matching can only happen where the geofence tag is store, but not parking lot.
+- **Use Radar's place geometry.** By default, if a place match is found, [circle geofences](/dashboard#circle) will adopt both the center and geometry of the Radar place. This behavior can be changed such that circle geofences only take on the center but not the polygon.
```
Comment (berylw) [inline docs/dashboard.mdx:86]: nit: should we use geometry to be consistent with the first sentence? `..circle geofences only take on the center but not the geometry`

File: docs/dashboard.mdx (hunk 2 above)
Comment (jsani-radar) [reply docs/dashboard.mdx:53]: yeah, this sentence is a bit of a run-on sentence, will rework

**Pull request merged**